apps/sms/style/sms.css

```diff
@@ -902,6 +902,18 @@ section[role="region"].new .article-list[data-type="list"]  li {
   pointer-events: none;
 }
 
+/* margin bottom adjustment for inline-block bdi element */
+.suggestion > .name {
+  margin-bottom: -0.5rem;
+}
+
+.suggestion > .name > bdi {
```

Comment:
Do you mean that we should reorder these RTL related styling to the bottom of the css file(because all of the RTL styling is placed at the bottom)?